Small fix and additions

crater2150 opened the pull request:
`class C<A extends Comparable, Serializable> {}` creates a class with 2 type parameters instead of restricting `A` to two interfaces.

Maybe leave out the commit adding the syntax for catching multiple exception types in one block, if you feel like it is too uncommon.

Commit:
Add default arguments

shawjef3 commented:
@crater2150  thanks for your suggestions! They look good except maybe the bean one. My understanding is that a bean must have a constructor that takes no arguments, but I looked at the output of `javap -c` and didn't see a no-arg constructor for `case class C(@BeanProperty var c: Int =0)`.

crater2150 commented:
You're right, it doesn't work. I forgot that the default arguments are not usable from Java. I see that most of the changes have already been added since I made the PR, so I removed everything except the default arguments one and rebased it.
